amplfi/architectures/pyproject.toml

```diff
@@ -2,14 +2,17 @@
 name = "amplfi.architectures"
 version = "0.1.0"
 description = "Architectures for amplfi"
-authors = ["Ethan Jacob Marx <ethan.marx@ligo.org>"]
+authors = ["Ethan Jacob Marx <ethan.marx@ligo.org>", "Deep Chatterjee <deep.chatterjee@ligo.org>"]
 packages = [{include = "amplfi"}]
 
 
 [tool.poetry.dependencies]
 python = ">=3.9,<3.12"
-pyro-ppl = "^1.9.0"
+ml4gw = {path = "../../ml4gw", develop = true}
+pyro-ppl = "*"
```

Comment:
Is there a reason we're unconstraining `pyro`? I imagine there's _some_ version where our code will break.

Reply:
Usually we should constrain a version when a feature breaks, and mention the issue from the corresponding repo. So far I have not found our pyro features to break.

Reply:
But at the same time, I don't feel very strongly.

Reply:
done